src/jira.js

```diff
@@ -696,6 +712,82 @@ export default class JiraApi {
     })));
   }
 
+   /** Add an option for a select list issue field.
+   * [Jira Doc](http://docs.atlassian.com/jira/REST/latest/#api/2/field/{fieldKey}/option-createOption)
+   * @name createFieldOption
+   * @function
+   * @param {string} fieldKey - the key of the select list field
+   * @param {object} option - properly formatted Option object
+   */
+  createFieldOption(fieldKey, option) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: `/field/${fieldKey}/option`,
+    }), {
+      method: 'POST',
+      followAllRedirects: true,
+      body: option,
+    }));
+  }
+
+  /** Returns all options defined for a select list issue field.
+   * [Jira Doc](http://docs.atlassian.com/jira/REST/latest/#api/2/field/{fieldKey}/option-getAllOptions)
+   * @name listFieldOptions
+   * @function
+   * @param {string} fieldKey - the key of the select list field
+   */
+  listFieldOptions(fieldKey) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: `/field/${fieldKey}/option`,
+    })));
+  }
+
+  /** Creates or updates an option for a select list issue field.
+   * [Jira Doc](http://docs.atlassian.com/jira/REST/latest/#api/2/field/{fieldKey}/option-putOption)
+   * @name upsertFieldOption
+   * @function
+   * @param {string} fieldKey - the key of the select list field
+   * @param {string} optionId - the id of the modified option
+   * @param {object} option - properly formatted Option object
+   */
+  upsertFieldOption(fieldKey, optionId, option) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: `/field/${fieldKey}/option/${optionId}`,
+    }), {
+      method: 'PUT',
+      followAllRedirects: true,
+      body: option,
+    }));
+  }
+
+  /** Returns an option for a select list issue field.
+   * [Jira Doc](http://docs.atlassian.com/jira/REST/latest/#api/2/field/{fieldKey}/option-getOption)
+   * @name getFieldOption
+   * @function
+   * @param {string} fieldKey - the key of the select list field
+   * @param {string} optionId - the id of the option
+   */
+  getFieldOption(fieldKey, optionId) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: `/field/${fieldKey}/option/${optionId}`,
+    })));
+  }
+
+  /** Deletes an option from a select list issue field.
+   * [Jira Doc](http://docs.atlassian.com/jira/REST/latest/#api/2/field/{fieldKey}/option-delete)
+   * @name deleteFieldOption
+   * @function
+   * @param {string} fieldKey - the key of the select list field
+   * @param {string} optionId - the id of the deleted option
+   */
+  deleteFieldOption(fieldKey, optionId) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: `/field/${fieldKey}/option/${optionId}`,
+    }), {
+      method: 'DELETE',
+      followAllRedirects: true,
+    }));
+  }
+
```

Comment:
I also don't see any unit tests.  At least add some tests to ensure these functions are hitting the expected URL's.